src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
+	private static final int STACK_EMPTY = -1;
+	private static final int STACK_FULL = 11;
+	private static final int STACK_CAPACITY = 12;
+	private int[] NUMBERS = new int[STACK_CAPACITY];
 
-	public int total = -1;
+	public int getTotal() {
+		return total;
+	}
+
+	private int total = STACK_EMPTY;
 
 	public void countIn(int in) {
 		if (!isFull())
 			NUMBERS[++total] = in;
 	}
 
 	public boolean callCheck() {
-		return total == -1;
+		return total == STACK_EMPTY;
 	}
 
 	public boolean isFull() {
-		return total == 11;
+		return total == STACK_FULL;
 	}
 
 	protected int peekaboo() {
 		if (callCheck())
-			return -1;
+			return STACK_EMPTY;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)